src/potemkin/collections.clj

```diff
@@ -118,8 +118,10 @@
   Object
   (hashCode [this]
     (reduce
-      (fn [acc [k v]]
-        (unchecked-add acc (bit-xor (.hashCode k) (.hashCode v))))
+     (fn [acc [k v]]
+       (unchecked-add acc (bit-xor
+                           (if k (.hashCode k) 0)
```

Comment:
Nit, maybe `(if (some? k) (.hashCode k) 0)`, since `(not= 0 (.hashCode false))`...